jest.config.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+module.exports = {
+  clearMocks: true,
+  collectCoverage: true,
+  coverageDirectory: "coverage",
+  coveragePathIgnorePatterns: [
+    "/node_modules/"
+  ],
```

Comment:
**question:** Just as a general question for these config files, I see that this flag takes `["/node_modules/"]` as default value. Now for flags which have our expected value as default value, like this case, should we still explicitly add this flag?

Reply:
0400d0e

Reply:
Generally not, I didn't know that this is also the default value :o Thanks for hinting that!